### ISSUE TITLE
Keydown Events not Working in IE11 for Space bar, Escape and Delete

### ISSUE THREAD

**Issue by nitramL** [vuejs/vue#9112]
### Version
2.5.17

### Reproduction link
[https://jsfiddle.net/cwnv6djs/](https://jsfiddle.net/cwnv6djs/)

### Steps to reproduce
Click on the input field
Press the space bar or Escape or Delete

### What is expected?
The label under the input should display "space" or "escape" or "delete"

### What is actually happening?
Nothing

---
Seems like KeyboardEvent.key in IE11 are :
- "Spacebar" for space bar
- "Esc" for Escape
- "Del" for Delete

**Comment by posva** [comment]
Escape was already fixed but maybe others need to be added here 
https://github.com/vuejs/vue/commit/1bd6196fb234c28754d9a27095afe0b5b84990ad. I haven't checked the repro yet though

**Comment by nitramL** [comment]
@posva you're right, Escape was fixed in 2.5.17-beta.0 (https://github.com/vuejs/vue/blob/v2.5.17-beta.0/src/compiler/codegen/events.js) but isn't in 2.5.17 (https://github.com/vuejs/vue/blob/v2.5.17/src/compiler/codegen/events.js)...

Wasn't 2.5.17-beta.0 released before 2.5.17 ?
If so, shouldn't this change be merged in 2.5.17 ?

**Comment by posva** [comment]
this will be released in next version, that last patch was a security release

**Comment by nitramL** [comment]
Crystal clear!
Thanks for the explanation :+1:
